packages/google_maps_flutter/example/test_driver/google_maps.dart

```diff
@@ -57,4 +57,43 @@ void main() {
     compassEnabled = await inspector.isCompassEnabled();
     expect(compassEnabled, true);
   });
+
+  test('testGetVisibleRegion', () async {
```

Comment:
I think we can make this test stronger by making sure the bounds are what we expect. I made a few changes at https://github.com/flutter/plugins/commit/36ad3891bdd31284a10112a59083626e3693e457 to make this test slightly better, if you agree with them, please make those changes.

Reply:
test with your changes not passed on my pixel 2 and nvidia shiesl tablet. seems the reason for the different screen. because of this I wrote not stronger test.

Reply:
Oh, interesting, I tried to make the tests unaware of the resolution. I made a mistake in the last 2 lines of the test, mind changing them to:

```
    expect(secondVisibleRegion.contains(southWest), isTrue);
    expect(secondVisibleRegion.contains(northEast), isTrue);
```

If they still fail. Could you paste the errors here? I can help you out.

Reply:
still error on line
expect(secondVisibleRegion.contains2(northEast), isTrue);. actual false

Reply:
I'm getting this error on Pixel 3 as well. I'm now looking at what is going on.

Reply:
Oops, I had illegal value for the latLng bounds before. They were outside of earth 😞

These should work well, could you please try these:

```
const LatLng southWest = LatLng(60, 75);
const LatLng northEast = LatLng(65, 80);
```

Reply:
very strange behavior, to test passed on tablet with portrait and landscape orientations need add next line, it is need for portrait orientation:
`    await Future<dynamic>.delayed(Duration(seconds: 3));`
after 
`    final GoogleMapController mapController =
    await mapControllerCompleter.future;`

and change line to, it is need for landscape orientation:
`    await mapController
        .moveCamera(CameraUpdate.newLatLngBounds(latLngBounds, 1));`

may be it specifically only for my device?

Reply:
can you push a commit with the latest changes, I will try to run it and see if I can reproduce this. Thanks for your patience @pepegich

Reply:
I pushed. I tested on another tablet Lenovo TAB 2 A10 70L. and test not passed without above workaround.